pkg/cloudprovider/aws/packing/nodecapacity.go

```diff
@@ -15,14 +15,17 @@ limitations under the License.
 package packing
 
 import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/awslabs/karpenter/pkg/utils/scheduling"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
 // TODO get this information from node-instance-selector
 var (
-	nodeCapacities = []*nodeCapacity{
+	fallbackNodeCapacities = []*nodeCapacity{
```

Comment:
I think we should remove this, I don't think "fallback" will work in practice (maybe this is just temporary?)